tests/integration/components/participant-types-test.js

```diff
@@ -3,24 +3,29 @@ import { setupRenderingTest } from 'ember-qunit';
 import { render } from '@ember/test-helpers';
 import hbs from 'htmlbars-inline-precompile';
 import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
-import seedMirage from '../../../mirage/scenarios/default';
 
 module('Integration | Component | participant-types', function(hooks) {
   setupRenderingTest(hooks);
   setupMirage(hooks);
 
   hooks.beforeEach(async function() {
     this.store = this.owner.lookup('service:store');
-    seedMirage(server);
   });
 
   test('it generates a list of participantTypes', async function(assert) {
-    // TODO: Retrieve by email?
-    this.user = await this.store.findRecord('user', 2, {
-      include: 'userProjectParticipantTypes.project,projects',
+    this.user = await this.store.createRecord('user');
+    this.project = await this.store.createRecord('project', { user: this.user });
```

Comment:
~Why store this on the function scope? If I'm not mistaken, the function scope is shared across other tests, which could cause memory issues.~ nm

Reply:
> @godfreyyeung this is looking great! One testing blindspot for us is the `show-project` page. I'll work on adding coverage there, but make sure you manually check that it still works in the development environment `ember s`. Thanks.

Yup, did a visual check and it looks okay! Ahh, thanks for helping with the test coverage on that page! 🙏